frontend/src/components/calendar/event-dialog/EventDialog.js

```diff
@@ -422,7 +334,7 @@ function EventDialog(props) {
         <span id="volunteer-sign-up">
           {" "}
           {/* Hack to get tooltip to display when button is disabled */}
-          <Button disabled={tooltipError} onClick={signUpUser}>
+          <Button disabled={tooltipError} variant="outline-primary" onClick={signUpUser}>
             Sign Up
           </Button>
```

Comment:
If Admin, prevent this button from being seen

Reply:
Wait, why should admins not be able to sign up for an event?

Reply:
Oh I guess I hadn't thought about it that way. But would they need to approve themselves? I also noticed that as a volunteer when I signed up my account went straight into approved rather than pending on the admin side. Not sure if that was intended or not? It might make it easier for an admin if that is the case though, so they don't have to approve every single time a volunteer signs up for something. Also, thoughts on email functionality when a user signs up for an event? Doesn't have to be this PR but that might be helpful, I'm not exactly sure how extensive that is. I can look into it.

Reply:
I could try this out, but I think that this is maybe desired rather than totally necessary [https://nodemailer.com/about/](url)

Reply:
Not sure if we want to send emails to them about it. That'd definitely be something to ask the Fin.

The current functionality for approval is:
- If you're an admin you automatically get approved for anything
- If you're using the default time slots for that event (if event is 10-12 and you sign up with hours 10-12), then you are automatically approved (can still be rejected by admins later)
- Pending is only for people who have signed up for hours not in the event's default time slot

Reply:
Ah okay that's perfect!!